compiler/options.nim

```diff
@@ -44,7 +44,7 @@ type                          # please make sure we have under 32 options
     optImportHidden
 
   TOptions* = set[TOption]
-  TGlobalOption* = enum       # **keep binary compatible**
```

Comment:
make sure you agree with removing this comment

Reply:
I do.